src/test/java/seedu/address/commons/util/StringUtilTest.java

```diff
@@ -123,6 +124,83 @@ public void containsWordIgnoreCase_validInputs_correctResult() {
         assertTrue(StringUtil.containsWordIgnoreCase("AAA bBb ccc  bbb", "bbB"));
     }
 
+    //---------------- Tests for wordsContainSubstringIgnoreCase --------------------------------------
```

Comment:
Excellent tests covering different equivalence partitions here. It can really minimise errors in the event any changes is made to the method in the future